Gruntfile.js

```diff
@@ -85,72 +23,58 @@ module.exports = function(grunt) {
     return file;
   });
 
-  var pluginCombinations = combine(plugins);
-  var pluginConcatFiles = pluginCombinations.reduce(function(dict, comb) {
-    var key = comb.map(function(plugin) {
-      return path.basename(plugin, '.js');
-    });
-    key.sort();
-
-    var dest = path.join('build/', key.join(','), '/raven.js');
-    dict[dest] = ['src/singleton.js'].concat(comb);
-
-    return dict;
-  }, {});
-
-  var browserifyConfig = {
-    options: {
-      banner: grunt.file.read('template/_copyright.js'),
-      browserifyOptions: {
-        standalone: 'Raven' // umd
-      },
-      transform: [versionify],
-      plugin: [derequire, collapser]
-    },
+  var rollupConfig = {
     core: {
-      src: 'src/singleton.js',
-      dest: 'build/raven.js'
+      options: [
+        {
+          input: {
+            input: 'src/singleton.js'
+          },
+          output: {
+            file: 'build/raven.js',
+            name: 'Raven',
+            banner: grunt.file.read('template/_copyright.js')
+          }
+        }
+      ]
     },
-    'plugins-combined': {
-      files: pluginConcatFiles,
-      options: {
-        transform: [[versionify], [new AddPluginBrowserifyTransformer()]]
-      }
+    plugins: {
+      options: []
     },
     test: {
-      src: 'test/**/*.test.js',
-      dest: 'build/raven.test.js',
-      options: {
-        browserifyOptions: {
-          debug: false // source maps
-        },
-        ignore: ['react-native'],
-        plugin: [proxyquire.plugin]
-      }
+      options: [
+        {
+          input: {
+            input: 'test/raven.test.js'
```

Comment:
It misses all plugins and integration tests

Reply:
all files are now generated.